montblanc/tests/beam_factory.py

```diff
@@ -25,7 +25,7 @@ def beam_factory(polarisation_type='linear',
                     frequency=None,
                     dtype=np.float64,
                     schema=DEFAULT_SCHEMA,
-                    overwrite=False):
+                    overwrite=True):
```

Comment:
Do we want to always overwrite?

Reply:
yup there is an issue with it not writing the beams otherwise